apps/web-giddh/src/app/daybook/daybook.component.html

```diff
@@ -38,15 +38,16 @@
                         </div>
                     </div>
                     <div class="form-inline d-block flex-row flex-nowrap daybook-filter">
-                        <div class="form-group custom-datepicker" (click)="showGiddhDatepicker($event)">
+                        <div class="form-group custom-datepicker mr-r15" (click)="showGiddhDatepicker($event)">
                             <i class="icon-calender calendar-addon"></i>
                             <input type="text" name="selectedDateRange"
                                 [value]="(selectedDateRangeUi) ? selectedDateRangeUi : ''"
                                 class="giddh-datepicker date-range-picker border-radius-2" />
                         </div>
-                        <div class="form-group cursor-pointer">
-                            <div class="advance-icon align-items-center"
+                        <div class="form-group cursor-pointer mobile-view float-right">
+                            <div class="advance-icon align-items-center mr-0 btn btn-primary"
                                 [tooltip]="commonLocaleData?.app_advance_search" (click)="onOpenAdvanceSearch()">
+                                <span class="pd-r05">Filter</span>
```

Comment:
Do add multi-lingual labels and avoid hard coded values

Reply:
done